### ISSUE TITLE
How can I generate the code?

### ISSUE THREAD

**Issue by hjJunior** [jaguar-orm/sqflite#10]
I can't run `pub run build_runner build` on Flutter projects, how can generate the code?

**Comment by tejainece** [comment]
Did you find the solution?

**Comment by hjJunior** [comment]
Yep, sorry, It was just add `flutter packages pub ...`

**Comment by wmorrison1967** [comment]
I am having the same issue.  I got this error at first :"Could not find a file named "pubspec.yaml" in "C:\Users\admin\AppData\Roaming\Pub\Cache\hosted\pub.dartlang.org\built_value-6.5.0"."
Once I added the "flutter packages" to the "pub run build_runner build" command the error went away.  But no code was generated.  This the content of my class:
class SystemSettings {
  SystemSettings();

  //SystemSettings.make();

  @PrimaryKey(auto: true)
  int _id;

  @Column(isNullable: true)
  int _setup_stage;

  @Column(isNullable: true)
  String _facebook_info;

  @Column(isNullable: true)
  String _twitter_info;
}
The bean code is not being generated.
